app/src/main/java/com/andryoga/safebox/ui/view/login/LoginFragment.kt

```diff
@@ -49,14 +49,12 @@ class LoginFragment : Fragment() {
 
             if (binding.ShowHintText.text == getString(R.string.show_hint)) {
```

Comment:
Good work with using correct methods but think about it, do you really need this if-else condition here? I think you can easily optimize this code and reduce next 8-9 lines of code to 2-3.

Reply:
okay